[ADD][8.0] Field product_id added on model mgmtsystem.nonconformity due to an ISO/TS 16949 requirement.

egrivero opened the pull request:
Due to an ISO/TS 16949 requirement when occurs a nonconformity for any failure that could result in a probable shipment of nonconforming product, the document nonconformity requires it can be associate a product, in response to this requirement the field product_id is added to the model mgmtsystem.nonconformity

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vauxoo/management-system/1)

<!-- Reviewable:end -->

nhomar commented:
@egrivero 

Can you format properly the information?
1. Shorter title. (max 80 chars)
2. Formated body.

Use github markdown for more information.
https://help.github.com/categories/writing-on-github/

You can achieve this directly for the commit.
http://stackoverflow.com/questions/29933349/how-can-i-make-git-commit-messages-divide-into-multiple-lines

With this guideline

**https://github.com/Islandora/islandora/wiki/Git-Guidelines-and-Best-Practices**

nhomar commented:
test x

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/vauxoo/management-system/1#-:-KFIUWtjqWFQt-3KrLZq:-1223497297)_

<!-- Sent from Reviewable.io -->

nhomar commented:
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/vauxoo/management-system/1)_

<!-- Sent from Reviewable.io -->

sabrinagnu commented:
@egrivero may you please fix the flake8 error?.

egrivero committed:
[ADD][8.0] Due to an ISO/TS 16949 requirement when occurs a nonconformity
for any failure that could result in a probable shipment of nonconforming
product, the document nonconformity requires it can be associate a product, in
response to this requirement the field product_id is added to the model
mgmtsystem.nonconformity

sabrinagnu commented:
@nhomar this LGTM, may you please merge?. Best regards!

sabrinagnu commented:
@nhomar FYI, the product is not required True, but that LGTM because on the ISO reading I see that the nonconformities not ALWAYS have a product related even if for Wohlert this should be requiered, I think that would be an improvement to be develop on Wohlert repository.
